deaf-and-hard-of-hearing/event-organizers/conference-best-practices.md

```diff
@@ -0,0 +1,141 @@
+
+#### Include interpreters in recording
+
+If you record your sessions, be sure the interpreters are on screen so deaf attendees can benefit from your accessibility effort even when watching them on-demand. 
+
+#### Global or national conferences 
+
+If you only have a few deaf individuals attending, provide two interpreters per individual (they always come in tandem for engagements longer than one hour). Wherever the deaf person goes, they'll follow. This will allow deaf individuals not only to attend sessions, but also network with their peers, walk the showfloor and talk with vendors. This is particularly important if they are the only or one of a few deaf people attending, as any interaction is contingent on the availability of an interpreter.
+
+Of course, this approach doesn't scale well. If you have more deaf attendees, you'll need to share accommodations among more individuals. 
+
+Before the conference, ask attendees to sign up for sessions in advance so you know which sessions must be staffed by interpreters. But allow for last-minute changes and be prepared to inform interpreters about these. While you'll want to plan ahead, you also want to offer attendees the flexibility to change their minds. Maybe they met someone who's giving a talk, and they'd like to attend that session. If you have multiple deaf attendees, have interpreters at keynotes by default. 
+
+##### Accommodation desk
+
+Set up an accommodation desk where attendees can ask questions, request changes, or get an interpreter spontaneously, if available. 
+
+##### Networking events
+
+As mentioned in the [conference networking opportunities section](#conference-networking-opportunities) below, networking is one of the key reasons people attend conferences, so be sure interpreters are also available during networking sessions. 
+
+When it comes to networking events outside of the official program (e.g., sponsor happy hours), try to find a way to fund interpreters, too. If you can't use the conference budget, consider creating a sponsorship add-on to fund networking interpreter services. Don't underestimate the importance of these valuable opportunities. 
+
+#### Regional/single-track conferences 
+
+It's easier to share accommodations among speakers and attendees at smaller or single-track conferences. Provide live captions and a pair of sign language interpreters.
+
+If your budget is tied, consider offering an **accessibility sponsorship add-on **and mention throughout the conference that interpreters are made possible thanks to that company. That will motivate them or other companies to participate next time. 
+
+#### Local and community events
+
+For local and community events where attendance is free or low-cost, we recommend offering an accessibility sponsorship add-on (see prior section) to try to have interpreters on site. If that doesn't work, live captions are certainly the minimum you should offer. Consider bringing a tablet as a backup. [any tech suggestions?]
```

Comment:
We need suggestions for tech that might help with communication.

Reply:
As a note - I use automatic captioning quite a lot, and I push my tools pretty hard. I can put together some thoughts on this, but perhaps it should be a separate document or added in a follow-up PR?

Depending on the automatic captioning tool, I believe some of them have T&Cs about not using them for public performance situations. I've been evaluating an OSS solution for Linux/Mac that, AFAIK, does not have that issue.

I think the Fedora design community did a virtual conference earlier this year that included some talks with captions, let me reach out to them and ask about it; they might have some ideas that would be useful for virtual/in-person conferences.

@JonZeolla I'm probably going to hit you up on Slack for info about the solution you're using, since I am trying to push one of our local tech conferences to do more on captioning too. :)

Reply:
Ok, should we just leave it at "If that doesn't work, live captions are certainly the minimum you should offer. Be sure to let the deaf attendee know before the event that you couldn't raise funds for interpreters and that, unfortunately, only captioning will be provided. Also, offer a refund if desired." 

 They should be apologetic about it. But it's a reality that they may not have the funds.